AppController/lib/haproxy.rb

```diff
@@ -407,12 +410,15 @@ def self.get_haproxy_stats(app_name)
 
       if service_name == "BACKEND"
         total_req_in_queue = parsed_info[REQ_IN_QUEUE_INDEX].to_i
+        current_sessions = parsed_info[CURRENT_SESSIONS_INDEX].to_i
         Djinn.log_debug("#{full_app_name} #{service_name} Queued Currently " +
           "#{total_req_in_queue}")
+        Djinn.log_debug("Current sessions for #{full_app_name} per AppServer " +
```

Comment:
You are right, I will change the log statement. We don't really need per AppServer and make the decision based on the total current sessions. Thanks for pointing that out!